src/hazelweb/gui/UndoHistoryPanel.re

```diff
@@ -1082,8 +1082,10 @@ let view = (~inject: Update.Action.t => Vdom.Event.t, model: Model.t) => {
       )
     );
 
-  let get_elt_id_under_mouse = _: option((int, int)) => {
-    let elt: Js.t(Dom_html.divElement) = JSUtil.unsafe_get_elt_under_mouse();
+  /* return option((group_id, elt_id)) */
```

Comment:
going to approve this one, but for your next clean up task can you define the types group_id and elt_id explicitly and then use them everywhere, instead of int?